src/javascripts/test/unit/Crud/fieldView/BooleanFieldView.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('chai').assert;
```

Comment:
useless test imo, but you should definitely test `maBooleanColumn` and `maBooleanField` is you're into adding more tests.